Content.Server/Chat/Systems/ChatSystem.Radio.cs

```diff
@@ -13,7 +13,33 @@ public sealed partial class ChatSystem
     /// Cache of the keycodes for faster lookup.
     /// </summary>
     private Dictionary<char, RadioChannelPrototype> _keyCodes = new();
-
+    //Dict of Headset IDs and what their "defaults" are
+    //probably should be put into the yml files themselves down the line
+    private Dictionary<string, string> headsetCommonDict = new()
+            {
+                { "ClothingHeadsetQM", ":u "},
+                { "ClothingHeadsetCargo", ":u "},
+                { "ClothingHeadsetAltCargo", ":u "},
+                { "ClothingHeadsetMining", ":u "},
+                { "ClothingHeadsetCentCom", ":y "},
+                { "ClothingHeadsetAltCentCom", ":y "},
+                { "ClothingHeadsetCommand", ":c "},
+                { "ClothingHeadsetAltCommand", ":c "},
+                { "ClothingHeadsetCE", ":e "},
+                { "ClothingHeadsetEngineering", ":e "},
+                { "ClothingHeadsetAltEngineering", ":e "},
+                { "ClothingHeadsetMedical", ":m "},
+                { "ClothingHeadsetAltMedical", ":m "},
+                { "ClothingHeadsetMedicalScience", ":m "},
+                { "ClothingHeadsetRD", ":n "},
+                { "ClothingHeadsetRobotics", ":n "},
+                { "ClothingHeadsetScience", ":n "},
+                { "ClothingHeadsetAltScience", ":n "},
+                { "ClothingHeadsetSecurity", ":s "},
+                { "ClothingHeadsetAltSecurity", ":s "},
+                { "ClothingHeadsetService", ":v "},
+                { "ClothingHeadsetAltSyndicate", ":t " },
+            };
```

Comment:
This needs to be done in yml.

Reply:
Made it so this is something specified in headsets' yml instead of a big dictionary/hardcoded